node/node_test.go

```diff
@@ -211,3 +211,16 @@ func TestWindowPostDispute(t *testing.T) {
 
 	test.TestWindowPostDispute(t, builder.MockSbBuilder, 2*time.Millisecond)
 }
+
+func TestWindowPostDisputeFails(t *testing.T) {
```

Comment:
In case anyone is wondering like me, yes, this gets run in the post-dispute CI -> https://circle-production-customer-artifacts.s3.amazonaws.com/picard/5d174a4f4cf8a70001f53d90/600b5238788ab961dcfe50e3-0-build/artifacts/tmp/test-artifacts/window-post-dispute.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20210125T153126Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20210125%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=490d8aa85bbe7d3816084b58c31dbfebfbb728804c6bc6edf132a25270962387

Reply:
Ah, yes... And the reason the `TestWindowPostDispute` and `TestWIndowedPost` don't run at the same time... is the `ed`.